examples/tryout/view_transform_2.cpp

```diff
@@ -0,0 +1,52 @@
+#include <gstorm.h>
+#include <vector>
+#include <iostream>
+#include <range/v3/all.hpp>
+
+#include "experimental.h"
+
+class TripleNum {
+  public:
+  constexpr TripleNum() {};
+  int operator()(int a) const {
+    return a*3;
+  }
+};
+
+int main() {
+
+  size_t vsize = 1024;
+
+  std::default_random_engine generator;
+  std::uniform_int_distribution<int> distribution(0,10);
+
+  auto generate_int =
+    [&generator, &distribution]() { return distribution(generator); };
+
+  std::vector<int> va(vsize);
+  std::vector<int> vb(vsize);
+  std::vector<int> vc(vsize);
+
+  ranges::generate(va, generate_int);
+  ranges::generate(vb, generate_int);
+
+  {
+    gstorm::sycl_exec exec;
+
+    auto ga = std::experimental::copy(exec, va);
+    auto gb = std::experimental::copy(exec, vb);
+    auto gc = std::experimental::copy(exec, vc);
+
+    std::experimental::transform(exec, ranges::view::transform(ga, gb, std::plus<int>{}), gc, TripleNum{});
+  }
+
+  auto expected = ranges::view::transform(va, vb, std::plus<int>{})
+    | ranges::view::transform(TripleNum{});
```

Comment:
Please indent `operator|` so it lines up with the `=` (this is the convention).

Reply:
is clang-format capable of formatting it properly?